pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
Well, it depends. If we want to have, let's say more usable URLs - then we should keep this ( also keep in mind that this a GET parameters not like a /session/<id>/ which is more like routing ). Current code allows us to store history in one place.
If I'll remake it to show only ?id= parameter, then it will require some changes ( moving breadcrumbs to View or sending state with filters values from View to Recording to form proper "go back" URL ).
What do you think?